package.json

```diff
@@ -63,7 +63,7 @@
   ],
   "devDependencies": {
     "@types/chai": "^4.2.14",
-    "@types/react-native": "^0.63.37",
+    "@types/react-native": "^0.64.13",
```

Comment:
Most recent @types/react-native is 13 days old - so older than 0.65 (which was released 7 days ago).
Hopefully there is no type changes in core react native classes.